draft-ietf-quic-transport.md

```diff
@@ -2784,8 +2784,11 @@ frame.  Its format is identical to the PATH_CHALLENGE frame
 
 If the content of a PATH_RESPONSE frame does not match the content of a
 PATH_CHALLENGE frame previously sent by the endpoint, the endpoint MAY generate
-a connection error of type UNSOLICITED_PATH_RESPONSE.
-
+a connection error of type UNSOLICITED_PATH_RESPONSE. Endpoints MUST NOT
+generate this error in response to a PATH_RESPONSE frame in a Handshake Packet.
```

Comment:
I don't think that we need to worry about this.  We have agreed not to specifically defend against tampering with the handshake.  Note that you need to be on-path for this to work, because of the handshake packet protection design.